wildlifelicensing/apps/main/forms.py

```diff
@@ -129,8 +137,11 @@ def __init__(self, *args, **kwargs):
     def clean(self):
         cleaned_data = super(IssueLicenceForm, self).clean()
 
-        if cleaned_data.get('end_date') < cleaned_data.get('start_date'):
-            raise forms.ValidationError('End date must be greater than start date')
+        end_date = cleaned_data.get('end_date')
+        start_date = cleaned_data.get('start_date')
+        if end_date and start_date and end_date < start_date:

```

Comment:
`is not None` for start and end date check, although you probably don't need this check because those fields are required and the required check would fail before this point.